ts-tests/tests/test-gas.ts

```diff
@@ -18,7 +18,7 @@ describeWithAcala("Acala RPC (Gas)", (context) => {
 			from: alice.getAddress(),
 			data: "0x" + Block.bytecode,
 		});
-		expect(gas.toNumber()).to.be.eq(11301014);
+		expect(gas.toNumber()).to.be.eq(11301114);
```

Comment:
we should convert those into snapshot tests at some point

Reply:
use `closeTo` instead